### PR TITLE
fix: fixes the debugging in the playground

### PR DESCRIPTION
**Pull request by Dominik1999**
Fixes mainly debugging

### CODE REVIEW

File: playground/src/pages/CodingEnvironment.tsx [hunk 1]
```diff
@@ -135,12 +137,15 @@ export default function CodingEnvironment(): JSX.Element {
       if (!debugExecutor) {
         throw new Error('debugExecutor is undefined');
       }
+      if (output == 'Debugging session started' && command == DebugCommand.Rewind) {
```
Comment (phklive) [inline playground/src/pages/CodingEnvironment.tsx:140]: Add a comment

File: playground/src/pages/CodingEnvironment.tsx [hunk 2]
```diff
@@ -175,10 +180,10 @@ export default function CodingEnvironment(): JSX.Element {
       return;
     }
 
-    if (value === 'advice_provider') {
+    if (value === 'advice_provider' || value === 'matrix_multiplication') {
```
Comment (phklive) [inline playground/src/pages/CodingEnvironment.tsx:183]: Add comments or generalise

**Review by phklive — APPROVED** [review]
Small nits regarding comments and generalisation of code.

File: playground/src/pages/CodingEnvironment.tsx (hunk 2 above)
Comment (Dominik1999) [reply playground/src/pages/CodingEnvironment.tsx]: changed it

**Pull request merged**